src/python/pants/backend/python/rules/pytest_coverage.py

```diff
@@ -212,7 +226,7 @@ def register_options(cls, register):
         register(
             "--report-output-path",
             type=str,
-            default=os.path.join(DistDir(relpath="dist").relpath, "coverage", "python"),
```

Comment:
`DistDir` wasn't behaving as intended. I think the intent was to use the `DistDir` configured in `pants.toml`, but that only works when you request the type as a singleton.

Here, `DistDir` was only a clunky wrapper around `PurePath`.